examples/domain-generalization-with-kd/README.md

```diff
@@ -32,13 +35,13 @@ python primeqa/mrc/run_mrc.py \
 ```
 This basic joint training functionality is offered through this source file so that one can train a teacher model first before distillation.
 
-The argument for `train_fof` in the command is a text file; each line in this file contains a path to a training json file. All training files must conform to the Hugging Face format for custom data files. The `eval_fof` argument works similarly. Below is an example of the content of such a file:
+The argument for `train_fof` in the command is a text file; each line in this file contains a path to a training json file. All training files must conform to the Hugging Face Datasets format for SQuAD-style data files. The `eval_fof` argument works similarly. Below is an example of the content of such a file:
```

Comment:
Perhaps you can include a link to the HF SQuAD datasets page?

Reply:
Done!